cmd/podman/common/volumes.go

```diff
@@ -88,17 +88,11 @@ func parseVolumes(volumeFlag, mountFlag, tmpfsFlag []string, addReadOnlyTmpfs bo
 			if _, ok := unifiedVolumes[dest]; ok {
 				continue
 			}
-			localOpts := options
-			if dest == "/run" {
-				localOpts = append(localOpts, "noexec", "size=65536k")
```

Comment:
I may have missed context here, but if I understand @TomSweeneyRedHat's suggestion, should that have been to remove only the `size=` portion, but not the `noexec`?

Reply:
I mentioned this in the PR

`Also /run is mounted on the host as exec, so no reason for us to mount
it noexec.`

Reply:
Oh, it's in the commit message. Sorry, I missed that.

/lgtm